Lib/idlelib/filelist.py

```diff
@@ -113,8 +113,10 @@ def canonize(self, filename):
 
 def _test():
     from idlelib.editor import fixwordbreaks
+    from idlelib.util import fix_scaling
```

Comment:
I don't care about _test either way.  It is old code that I do not use.  I don't even know if it runs.